app/ai/text/tests/test_blueprint.py

```
@@ -0,0 +1,473 @@
+                failed += 1
+                errors.append((test_name, str(e)))
+                print(f"  FAIL  {test_name}: {e}")
+            except Exception as e:
+                failed += 1
+                errors.append((test_name, traceback.format_exc()))
+                print(f"  ERROR {test_name}: {e}")
+
+    print(f"\n{'='*60}")
+    print(f"Results: {passed}/{total} passed, {failed} failed")
+    if errors:
+        print(f"\nFailed tests:")
```

Ruff (0.15.2)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)